### PR TITLE
Update fastlane plugin for sentry and bundler version

### PR DESCRIPTION
**Pull request by joshuatbrown**
## Issues covered
None

## Description
There's a message in the [logs](https://github.com/planetary-social/nos/actions/runs/10100747038/job/27932798302) from our latest build that says:

```
+----------------------------------------+
|        Plugin updates available        |
+--------+--------------+----------------+
| Plugin | Your Version | Latest Version |
+--------+--------------+----------------+
| sentry | 1.17.0       | 1.24.0         |
+--------+--------------+----------------+
[20:29:39]: To update all plugins, just run
[20:29:39]: $ bundle exec fastlane update_plugins
```

So after installing [mise](https://mise.jdx.dev) and the latest version of ruby, I was able to run `update_plugins`. Then it told me to run `bundle update fastlane` to update fastlane, so I did.

## How to test
I'm not sure. This is the fastlane sentry plugin, so I imagine you'd have to deploy and then see if data still appears in Sentry. So, I guess:

1. Deploy a build
2. Make sure Sentry still works

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -57,15 +60,15 @@ GEM
...
     faraday-net_http_persistent (1.2.0)
     faraday-patron (1.0.0)
     faraday-rack (1.0.0)
     faraday-retry (1.0.3)
     faraday_middleware (1.2.0)
       faraday (~> 1.0)
-    fastimage (2.2.7)
-    fastlane (2.217.0)
+    fastimage (2.3.1)
+    fastlane (2.218.0)
```
Comment (joshuatbrown) [inline Gemfile.lock:71]: This is the part I don't get. When I run `bundle exec fastlane -v`, it tells me I'm running 2.218.0 (which matches this) but then it also says 2.221.1 is available and to run `bundle update fastlane`. But then I run that again and it continues to stay on 2.218.0. 🤷

Comment (mplorentz) [reply Gemfile.lock:71]: 🤔 I updated ruby to 3.2.2 and was able to `bundle update fastlane` to 2.221.1. Should I push the changes and see what happens on your machine?

Comment (mplorentz) [reply Gemfile.lock:71]: I also put `gem 'fastlane', '~> 2.221'` in the Gemfile.

Comment (joshuatbrown) [reply Gemfile.lock:71]: Yeah, you can go ahead and push. Thanks

Comment (joshuatbrown) [reply Gemfile.lock:71]: The weird thing is that I'm on ruby 3.3.4. Who knows -- doesn't seem worth it to me to try to debug.

Comment (mplorentz) [reply Gemfile.lock:71]: oh 🤔 , 3.2.2 was the latest version listed in `rbenv` which is what I use to install ruby versions.

Comment (mplorentz) [reply Gemfile.lock]: I pushed and added a .ruby-version file which is picked up by some ruby tools and it seems good to have around.

**Review by mplorentz — APPROVED** [review]

**Pull request merged**